### service to query ptz status.
int32 xmin
int32 xmax
int32 ymin
int32 ymax
---
bool success
